### DISCUSSION TITLE
Question about "disabled_server_names"

### DISCUSSION THREAD

**Discussion by vladns** [DNSCrypt/dnscrypt-proxy#2499] [Q&A]
I want to use blacklist for dns, loading the list by url. In "disabled_server_names" you need to list the names, you can enter a link, but it doesn't seem to work. Here is this link:
https://raw.githubusercontent.com/kkkgo/PaoPao-Pref/main/dnscrypt_resolver/ban_list.txt

**Comment by lifenjoiner** [comment]
That does not follow the [config syntax](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Configuration-Sources#disabling-servers-by-name).
You must transform it following the syntax. It is also what others do.

**Comment by vladns** [comment]
I know that. But entering such a long list is not very convenient. Will a local file with the same syntax be accepted? Or do you still need the server names?

**Comment by lifenjoiner** [comment]
[Using a script is easy](https://github.com/kkkgo/PaoPaoDNS/blob/cf38234ec4338b25cc5eefb08b567003d61c33f7/src/build.sh#L62-L71). See how the list author uses it.

**Comment by vladns** [comment]
Okay, I understand your position, although I don’t share editing the config in this way. I would prefer a link to the file and periodic checking of this file by dnscrypt itself.

**Comment by lifenjoiner** [comment]
The public servers are verified and updated at https://github.com/DNSCrypt/dnscrypt-resolvers.
Even the `blocked_names` file is transformed from sources by a script, and "static".
I don't think `disabled_server_names` would be prioritized over `blocked_names`. The list is very subjective, and is a rare case.
If there is any critical issues, report to the servers repository, ping the server maintainer.
